tests/e2e/test_end_to_end.py

```diff
@@ -285,6 +285,7 @@ def test_cmip5_metacat_vals_found(metacat, colname, expected):
 
 
 @e2e
+@pytest.mark.xfail
```

Comment:
How well does the system handle the `xpass` generated if this happens to succeed?